src/plugins/notebook/components/NotebookEmbed.vue

```diff
@@ -141,6 +141,12 @@ export default {
                 });
         },
         changeLocation() {
+            if (this.openmct.editor.editing) {
+                this.openmct.notifications.error('Error navigating while in edit mode');
```

Comment:
Agree to a separate issue for preview mode, but I don't think we can leave it in its current state. Providing a link to the user that just shows an error when they click it makes no sense to me. It should not appear clickable at all. Either the cursor should be changed in edit mode (and the navigate event suppressed), or [the anchor tag should be conditional](https://github.com/nasa/openmct/pull/4091/files/7760e2786343a74c04675b31aef0f1ee97bfed55..a9763621fa7be55aaedd39f9693a2a20acc900fb#diff-ffd00ef23f895ed371e9a17c535e6bbef22815de5f39de2739659820f900a45bL11).

Reply:
removed changes, need to fix in #4115